packages/lesswrong/components/messaging/InboxWrapper.tsx

```diff
@@ -2,17 +2,42 @@ import React from 'react';
 import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useLocation } from '../../lib/routeUtil';
 import { useCurrentUser } from '../common/withUser';
+import { isEAForum } from '../../lib/instanceSettings';
+
+export type InboxComponentProps = {
+  terms: ConversationsViewTerms;
+  currentUser: UsersCurrent;
+  title?: JSX.Element | String;
+  isModInbox?: boolean;
+  classes: ClassesType;
+};
 
 const InboxWrapper = () => {
```

Comment:
This is the top level component that routes to the old or new version depending on whether this is LW or EAF. For EAF this is used for both `/inbox` and `/inbox/:_id` routes. For LW it's just used for `/inbox`